config/serverless.oblt.yml

```diff
@@ -141,11 +51,6 @@ xpack.fleet.agentIdVerificationEnabled: false
 ## Enable event.ingested separately because agentIdVerification is disabled
 xpack.fleet.eventIngestedEnabled: true
 
-## Enable the capability for the observability feature ID in the serverless environment to take ownership of the rules.
-## The value need to be a featureId observability Or stackAlerts Or siem
-xpack.alerting.rules.overwriteProducer: 'observability'
```

Comment:
Remove the overwriting producer. Producers should now match their values in stateful. 

However, I do still have some confusion about this. The producer value of the `Custom Threshold` rule is still set to `observability` somewhere in the code base, based on my testing. All other producers appear to match their stateful values.